src/diffusers/loaders/unet_loader_utils.py

```diff
@@ -108,21 +115,28 @@ def _maybe_expand_lora_scales_for_one_adapter(
     scales = copy.deepcopy(scales)
 
     if "mid" not in scales:
-        scales["mid"] = 1
+        scales["mid"] = default_scale
```

Comment:
I think it's okay to not use another function argument just to introduce a default value which is fairly well-known. So, I would remove `default_scale` and just keep 1.0 as before.